service/storage_service.cc

```diff
@@ -918,7 +919,8 @@ void storage_service::bootstrap() {
             { gms::application_state::CDC_STREAMS_TIMESTAMP, versioned_value::cdc_streams_timestamp(_cdc_streams_ts) },
             { gms::application_state::STATUS, versioned_value::bootstrapping(_bootstrap_tokens) },
         }).get();
-
+        set_mode(mode::JOINING, "Coming out of the shadow");
```

Comment:
Why only here and not in all places we come out of shadow?

Reply:
Other places have had other log messages nearby so adding this seemed unnecessary. But we could do it for consistency.